modules/candidate_parameters/ajax/getData.php

```diff
@@ -454,3 +455,100 @@ function getConsentStatusHistory($candID, $consents)
 
     return $commentHistory;
 }
+
+/**
+ * Handles the fetching of Consent Status fields
+ *
+ * @throws DatabaseException
+ *
+ * @return array
+ */
+function getMonitoringFields()
+{
+    $candID = $_GET['candID'];
+
+    $db        =& Database::singleton();
+    $candidate =& Candidate::singleton($candID);
+
+    // get pscid
+    $pscid = $db->pselectOne(
+        'SELECT PSCID FROM candidate where CandID = :candid',
+        array('candid' => $candID)
+    );
+
+    //get vists for this candidate
+    $visit_labels = $candidate->getListOfVisitLabels();
+
+    $flagged       =[];
```

Comment:
Spacing + probably should convert to `array()` since Dave is adding a new PHPCS rule